e2e-tests/test_config.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2019 The Ground Authors.
```

Comment:
=> 2024 here and throughout on new files.

Reply:
Sorry! Missed this in the sea of comments (Github decided to helpfully hide it) :P